models/tornado_cash/tornado_cash_deposits.sql

```diff
@@ -35,13 +35,13 @@ FROM
                 AND et.block_time >= (select min(evt_block_time) from {{ source('tornado_cash_ethereum','eth_evt_Deposit') }})
                 {% endif %}
                 {% if is_incremental() %}
-                AND et.block_time >= (select max(block_time) from {{ this }})
+                AND et.block_time >= date_trunc("day", now() - interval '1 week')
```

Comment:
Shouldn't the `LEFT JOIN` above be an `INNER JOIN` instead? And then you could drop the where filter below.

Reply:
added this common check to review doc, but many existing queries from v1 will have left joins to look out for

Reply:
There are cases where you'd want to do a left join as I discovered [here](https://github.com/duneanalytics/spellbook/pull/1465/files#diff-5bf74bdf14e57e88e2a0385fed61e5f0a5ba0d554f05f56bf255e413ca9eec7aR86). But generally it seems like it should be inner.